get-started/PyG-SchNetGNN.ipynb

```diff
@@ -123,32 +148,68 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "(Data(x=[5, 11], edge_index=[2, 8], edge_attr=[8, 4], y=[1, 19], pos=[5, 3], idx=[1], name='gdb_1', z=[5]),\n",
-       " tensor([6, 1, 1, 1, 1]),\n",
-       " tensor([[-1.2700e-02,  1.0858e+00,  8.0000e-03],\n",
-       "         [ 2.2000e-03, -6.0000e-03,  2.0000e-03],\n",
-       "         [ 1.0117e+00,  1.4638e+00,  3.0000e-04],\n",
-       "         [-5.4080e-01,  1.4475e+00, -8.7660e-01],\n",
-       "         [-5.2380e-01,  1.4379e+00,  9.0640e-01]]),\n",
-       " tensor([13.7363]))"
+       "(Data(x=[13, 11], edge_index=[2, 28], edge_attr=[28, 4], y=[1, 19], pos=[13, 3], idx=[1], name='gdb_125563', z=[13]),\n",
+       " tensor([6, 6, 7, 7, 6, 7, 7, 7, 7, 1, 1, 1, 1]),\n",
+       " tensor([[-2.7500e-02,  1.4963e+00,  5.2800e-02],\n",
+       "         [-9.1000e-03,  1.2800e-02,  2.6000e-03],\n",
+       "         [-4.2200e-02, -7.5060e-01, -1.0686e+00],\n",
+       "         [-9.3000e-03, -2.1018e+00, -7.2150e-01],\n",
+       "         [ 4.3600e-02, -2.0859e+00,  5.8330e-01],\n",
+       "         [ 1.0010e-01, -2.8658e+00,  1.7010e+00],\n",
+       "         [ 1.3480e-01, -2.0775e+00,  2.8101e+00],\n",
+       "         [ 1.0380e-01, -8.4570e-01,  2.4693e+00],\n",
+       "         [ 4.7300e-02, -8.2400e-01,  1.1011e+00],\n",
+       "         [ 8.7460e-01,  1.8923e+00,  5.3110e-01],\n",
+       "         [-8.0800e-02,  1.8742e+00, -9.6890e-01],\n",
+       "         [-8.9120e-01,  1.8675e+00,  6.1440e-01],\n",
+       "         [ 1.1880e-01, -3.8660e+00,  1.7999e+00]]),\n",
+       " tensor([5.2708]))"
       ]
      },
-     "execution_count": 4,
+     "execution_count": 5,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "datum = dataset[0]\n",
+    "datum = dataset[123244]\n",
     "datum, datum.z, datum.pos, datum.y[:, 4]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can apply a transform to the QM9 dataset that selects the properties we need for training SchNet"
```

Comment:
```suggestion
    "Using a transform to the QM9 dataset we can select the properties we need for training SchNet and convert them to a PyG Data object."
```
Sounded like we were doing more transforming of the data than just selecting + turning into the correct type

Reply:
thanks your version reads much easier